packages/branch-names/src/branches.ts

```diff
@@ -0,0 +1,63 @@
+import { GitHubActionsContext } from "@octopusdeploy/shared-action-utils";
+
+export function getBranchNames(context: GitHubActionsContext) {
+    try {
+        let isDefault: boolean = false;
+        let isTag: boolean = false;
+        let defaultBranch: string = "";
+        let currentBranch: string = "";
+        let baseRefBranch: string = "";
+        let headRefBranch: string = "";
+        let refBranch: string = "";
+        let tag: string = "";
+
+        const githubRef = context.getInput("gitHubRef");
+        const githubBaseRef = context.getInput("gitHubBaseRef");
+        const githubHeadRef = context.getInput("gitHubHeadRef");
+        const githubEventBaseRef = context.getInput("gitHubEventBaseRef");
+        const githubEventName = context.getInput("gitHubEventName");
+        const githubEventRepositoryDefaultBranch = context.getInput("gitHubEventRepositoryDefaultBranch");
+        const tagPrefixToRemove = context.getInput("tagPrefixToRemove");
```

Comment:
The outputs of our action are in a format `word_word_word`, I wonder if its worth making the inputs use the same format?

Reply:
Done